autometa/validation/datasets.py

```diff
@@ -0,0 +1,212 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+COPYRIGHT
+Copyright 2021 Ian J. Miller, Evan R. Rees, Kyle Wolf, Siddharth Uppal,
+Shaurya Chanana, Izaak Miller, Jason C. Kwan
+
+This file is part of Autometa.
+
+Autometa is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+Autometa is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with Autometa. If not, see <http://www.gnu.org/licenses/>.
+COPYRIGHT
+
+pulling data from google drive dataset with simulated or synthetic communities
+"""
+
+
+import gdown
+import os
+import sys
+import logging
+import pandas as pd
+
+from autometa.common.utilities import internet_is_connected
+
+logger = logging.getLogger(__name__)
+
+
+def download(type: str, size: list, file_names: list, dir_path: str) -> None:
+
+    """Downloads the files specified in a dictionary.
+
+    Parameters
+    ----------
+    type : str
+        specifies the type of dataset that the user would like to download from
+    size : list
+        specifies the size of dataset that the user would like to download
+    file_names : list
+        specifies the file(s) that the user would like to download
+    dir_path : str
+        dir_path_size path where the user wants to download the file(s)
+
+    Returns
+    -------
+    None
+        download is completed through gdown
+
+    """
+
+    if type == "synthetic" or type == "all":
+        sys.exit("Haven't implemented that yet")
+
+    for community_size in size:
+        index = pd.read_csv("gdown_fileIDs.csv", dtype=str)
+        index = index.query(f'dataset == "{community_size}"')
+        dir_path_size = ""
+        dir_path_size = os.path.join(dir_path, community_size)
+        # make a new directory
+        if not os.path.exists(dir_path_size):
```

Comment:
Need to make sure the provided `dir_path` exists or mkdir will fail

Reply:
Good catch! I added a logger error in main().